extensions/stackdriver/common/utils.cc

```diff
@@ -106,9 +107,12 @@ std::string getGCEInstanceUID(const ::Wasm::Common::FlatNode &node) {
   }
 
   if (name.size() > 0 && project && location) {
-    return absl::StrCat("//compute.googleapis.com/projects/",
-                        project->value()->string_view(), "/zones/",
-                        location->value()->string_view(), "/instances/", name);
+    return absl::StrCat(
+        "//compute.googleapis.com/projects/",
+        ::Wasm::Common::toAbslStringView(project->value()->string_view()),
```

Comment:
nit: doesn't absl StrCat handle std::string_view natively?

Reply:
It doesn't - it supports `absl::string_view`, which by default is equivalent to `std::string_view` in C++17, but Envoy switched to non-standard null-terminated variants in envoyproxy/envoy#16029, so `std::string_view` is no longer supported.

Reply:
@PiotrSikora Could you clarify wrt null?
is it null-terminated or nullptr constructed?

Reply:
Good catch - `nullptr`-constructed.